src/realtime/extension.ts

```diff
@@ -23,7 +24,7 @@ export class WebSocketWrapper implements ISocketWrapper {
 
   constructor() {
     if (typeof chrome.runtime === 'undefined')
-      throw new Error(
+      throw new SpeechmaticsInternalError(
```

Comment:
I think this should be an UnsupportedEnvironment error with a custom message, same for the window and node checks

Reply:
This has been updated